src/mcts/search.h

```diff
@@ -192,8 +192,10 @@ class Search {
 // within one thread, have to split into stages.
 class SearchWorker {
  public:
-  SearchWorker(Search* search, const SearchParams& params)
-      : search_(search), history_(search_->played_history_), params_(params) {}
+  SearchWorker(Search* search)
+      : search_(search),
+        history_(search_->played_history_),
```

Comment:
For consistency I would make this `search->played_history_`

Reply:
`history_` is a reference so sometimes it's important from where to initialize.
But here indeed as `search` is a pointer, it's fine.

Changed!